src/P5Functions/touchmouse.js

```diff
@@ -315,6 +315,8 @@ module.exports = function withTouch(p, curElement, attachEventHandler, document,
 
     if (delta && typeof p.mouseScrolled === 'function') {
       p.mouseScrolled();
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
@MatthiasSaihttam It will better to first inform the browser to stop propagation and then enter the `p.mouseScrolled` function, so can you move these two up to happen first? With that this should be fine to merge in.